assets/vue/views/documents/CreateFile.vue

```diff
@@ -34,24 +34,17 @@
 </template>
 
 <script>
-import { mapActions } from "vuex"
-import { createHelpers } from "vuex-map-fields"
 import DocumentsForm from "../../components/documents/FormNewDocument.vue"
 import Loading from "../../components/Loading.vue"
 import Toolbar from "../../components/Toolbar.vue"
 import CreateMixin from "../../mixins/CreateMixin"
 import { RESOURCE_LINK_PUBLISHED } from "../../components/resource_links/visibility"
 import Panel from "primevue/panel"
 import TemplateList from "../../components/documents/TemplateList.vue"
-import axios from "axios"
+import documentsService from "../../services/documents"
 
 const servicePrefix = "Documents"
 
-const { mapFields } = createHelpers({
-  getterType: "documents/getField",
-  mutationType: "documents/updateField",
-})
-
 export default {
   name: "DocumentsCreateFile",
   servicePrefix,
```

Comment:
What is the purpose of this line `servicePrefix`? It's defined above `const servicePrefix = "Documents"`

Reply:
Not sure, to be honest.